index.js

```diff
@@ -44,10 +44,10 @@ const plugin = (fastify, opts, next) => {
 
     // populating pathRegex if missing
     route.pathRegex = undefined === route.pathRegex ? opts.pathRegex : String(route.pathRegex)
-
+    bodyLimit = (route.bodyLimit) ? {  bodyLimit: route.bodyLimit } : null;
```

Comment:
Not sure about the reason for this statement?

Reply:
@jkyberneees I added that statement to keep the parameters clean, i actually missed something in first commit which later added to second commit. please have a look